airflow/providers/openlineage/extractors/base.py

```diff
@@ -35,6 +37,22 @@ class OperatorLineage:
     run_facets: dict[str, BaseFacet] = Factory(dict)
     job_facets: dict[str, BaseFacet] = Factory(dict)
 
+    @staticmethod
+    def merge(base: OperatorLineage, other: OperatorLineage) -> OperatorLineage:
+        """
```

Comment:
Should this be a separate static method? Shouldn't this be done while populating the lineage?

Reply:
@sunank200 it comes from commit in #31398 that this PR is dependent on. I've deleted this method though there.